python/lsst/meas/algorithms/loadReferenceObjects.py

```diff
@@ -1213,10 +1211,11 @@ def applyProperMotionsImpl(log, catalog, epoch):
         record.set(coordKey,
                    record.get(coordKey).offset(bearing=bearingRad*geom.radians,
                                                amount=amountRad*geom.radians))
+    # TODO: this needs to incorporate the full covariance!
```

Comment:
This should say `TODO DM-36979:`, since that's the ticket for that work. We wanted to put that off so that we could get this into review and make the catalog, then work on implementing the math and tests later.